doc/api/http2.md

```diff
@@ -195,9 +195,10 @@ event is emitted.
 added: v8.4.0
 -->
 
+* `settings` {Object} A copy of the `SETTINGS` frame received.
```

Comment:
I wonder if this should be the [`{HTTP/2 Settings Object}`](https://github.com/nodejs/node/blob/a4b4854e2ff0f3de69caa5e46d001575b9e22273/tools/doc/type-parser.js#L76) as in other sections of this doc.

Reply:
It should! Didn't know that type existed, now I need to write the whole js file. 😛